### PR TITLE
Support for Python2.6 (without argparse tests)

### LINKED ISSUE #17: Python 2.6 support
Would it be feasible to support Python 2.6 or is it a no go? Failing build log: https://travis-ci.org/letsencrypt/lets-encrypt-preview/jobs/60949344#L1366

Backref: https://github.com/letsencrypt/lets-encrypt-preview/pull/376

### PR DESCRIPTION
**Pull request by kuba**
partially fixes #17

### CODE REVIEW

**Comment by bw2** [comment]
I've tried merging this pull request with the latest version of master and fixing python2.6 support on this branch:
https://github.com/bw2/ConfigArgParse/tree/python26_support

When I run tox or `python2.6 setup.py test` though, I get `AttributeError: 'module' object has no attribute 'test_configargparse'`. Do you think you could fix/update this pull request so I can merge it?

**Comment by kuba** [comment]
Sorry, @bw2, but I've just seen your reply. Absolutely, feel free add new commits on top of this PR. However, I've merged it with the latest `master`, so you don't have to do it yourself anymore :)

Notably, I removed `configargparse.__version__`, so if you need it back, then I would suggest using `pkg_resources` for establishing currently installed package version, instead of duplicating version string. Let me know if you need it in your API, and I can send a relevant commit.

**Comment by kuba** [comment]
Friendly ping :)

**Comment by kuba** [comment]
@bw2, could you please merge this in?

**Comment by kuba** [comment]
@bw2, are you still maintaining this project?

**Comment by kuba** [comment]
FYI this is blocking Let's Encrypt support for Python2.6.

**Comment by jayvdb** [comment]
I am looking at using ConfigArgParse, but need Python 2.6 (unofficial) support.  happy to help maintain Python 2.6 compatibility if this is merged.

**Comment by kuba** [comment]
@bw2, should we consider this as a dead project?

File: setup.py [hunk 1]
```diff
@@ -69,9 +68,18 @@ def launch_http_server(directory):
...
+    install_requires.extend([
+        'argparse',
+        'ordereddict',
+        'unittest2',
+    ])
+
+
 setup(
     name='ConfigArgParse',
-    version=configargparse.__version__,
```
Comment (jayvdb) [inline setup.py]: You could read the `__init__` file, like I do in https://github.com/jayvdb/travis_log_fetch/blob/7c3e6925f77579d3d4e0c73d8e8ccb0151714150/setup.py#L57

**Comment by jayvdb** [comment]
Could you add Python 2.6 to `.travis.yml` ?

**Comment by bw2** [comment]
@kuba and @jayvdb - sorry it took this long. Hope this merge is still useful to you guys.

**Pull request merged**